x-pack/plugins/lists/server/routes/index.ts

```diff
@@ -44,3 +44,6 @@ export * from './update_exception_list_route';
 export * from './update_list_item_route';
 export * from './update_list_route';
 export * from './utils';
+
+// internal
+export * from './internal/create_exceptions_list_route';
```

Comment:
Should we rename to something like `create_internal_exceptions_list_route`?

Reply:
It is already inside the `internal` folder. Do you think it is still needed? I can do the change if so 🙂

Reply:
No, you're right!